DataFormats/Detectors/CTP/src/Scalers.cxx

```diff
@@ -361,3 +361,8 @@ int CTPRunScalers::updateOverflows(const CTPScalerRaw& scal0, const CTPScalerRaw
   //std::cout << std::endl;
   return 0;
 }
+int CTPRunScalers::parseZMQScalers(std::string zmqscalers)
+{
+  std::vector<std::string> tokens = o2::utils::Str::tokenize(zmqscalers, ' ');
```

Comment:
this method does nothing, just creates/destroys a local vector. Is this intended?